### DISCUSSION TITLE
Is there any doc about SSO?

### DISCUSSION THREAD

**Discussion by baominwang** [ory/.github#70] [General]
I try to find SSO related information but failed. Would you share with me some docs?

**Comment by baominwang** [comment]
When I debug the tutorial, I find that after the authentication, there is a cookie named ory_hydra_session_dev. I have two questions:
1. Is ory_hydra_session_dev used as SSO Global Session?
2. If the answer for #1 is yes, does it mean that Identity provider should implement SSO functionalities?

**Comment by vinckr** [comment]
Hello @baominwang 
can you elaborate a bit on the use case please? What do you mean by SSO exactly? 

As for your second question, what tutorial are you looking for? 

If you are looking to implement "Login with X" or passswordless login in your application I suggest to check out the [Ory Kratos Identity server](https://www.ory.sh/docs/identities/native-browser), that implements all the self-service flows, authentication methods, and user management that is missing in Ory Hydra.

**Reply by baominwang** [reply]
Our use case is as follows:
1. We have our own identity server hat facilitates user management and authentication.
2. We aim to utilize Ory Hydra for supporting Single Sign-On (SSO) across our three applications. The expected behavior is that when a customer signs in to application1 using their browser, Ory Hydra will establish an SSO session as a cookie. Subsequently, if the same customer attempts to access application2 with the same browser, an authorization request will be sent to Ory Hydra along with the cookie containing the SSO session information. Upon verifying that the session is still valid, Ory Hydra will directly return ID Token/Access Token without requiring any additional authentication.

**Reply by vinckr** [reply]
So SSO means for you "I log in on one website, and I am automatically logged on on some other websites", correct @baominwang ?

**Reply by baominwang** [reply]
Yes. That's true. The feature is supported by KeyCloak, Curity. I want to check whether it is supported by Ory Hydra with our own identity server.

**Reply by baominwang** [reply]
The sequence flow is just like https://curity.io/resources/learn/web-client-sso-with-openid-connect/

**Reply by jgrantprog1993** [reply]
@vinckr - Any updates on this ? I am looking for the same information
